.editorconfig

```diff
@@ -37,7 +28,28 @@ indent_size = 2
 [*.json]
 indent_size = 2
 
-# CSharp code style settings:
-[*.{cs}]
+# Scripting files
+[*.{ps1,bat,cmd}]
+indent_size = 4
+
+# Java code files
+[*.java]
+indent_size = 2
+
+# Pom files
+[pom.xml]
+indent_size = 2
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Coding styles
+# ----------------------------------------------------------------------------------------------------------------------
+
+# Dotnet code style settings:
+[*.{cs,vb}]
 # Sort using and Import directives with System.* appearing first
 dotnet_sort_system_directives_first = true
+# Avoid "this." and "Me." if not necessary
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
```

Comment:
What about `var`? (troll)

Reply:
Sadly it doesn't work with VS 2015, I did planned to add this discreetly...